apps/frontend/src/index.tsx

```diff
@@ -38,6 +39,7 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							},
 						],
 						element: <App />,
+						errorElement: <NotFound />,
```

Comment:
Instead add a new route for path "*"

Reply:
<img width="1044" alt="image" src="https://github.com/user-attachments/assets/bd8dcb46-9516-4d71-b283-4e893f4befec">

I added Not Found like this. Is it ok that i see navigation?
``` {
								element: <NotFound />,
								path: "*",
							},
```

Reply:
No, you shouldn't see the navigation